operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -60,6 +60,29 @@ type CentralSpec struct {
 	// Overlays
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Overlays,order=8,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
 	Overlays []*K8sObjectOverlay `json:"overlays,omitempty"`
+
+	// Monitoring configuration.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=9,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
+	Monitoring *GlobalMonitoring `json:"monitoring,omitempty"`
+}
+
+// GlobalMonitoring defines settings related to monitoring.
+type GlobalMonitoring struct {
+	OpenShiftMonitoring *OpenShiftMonitoring `json:"openshift,omitempty"`
+}
+
+// OpenShiftMonitoring defines settings related to OpenShift Monitoring
+type OpenShiftMonitoring struct {
+	//+kubebuilder:validation:Default=true
+	//+kubebuilder:default=false
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
Why use a boolean pointer rather than a boolean ?

Reply:
I've looked at other struct and tried to be consistent. Do you think we can have value here?